File: TourApp/data/src/main/java/com/example/data/database/StageDatabaseFactory.java
package com.example.data.database;

import android.content.Context;

import androidx.room.Room;

public class StageDatabaseFactory {

    public static StageDatabase getDatabase(Context context){
        return Room.databaseBuilder(context, StageDatabase.class, "db-contacts")
                .allowMainThreadQueries()
                .fallbackToDestructiveMigration()
                .build();
    }

}

File: TourApp/app/src/main/java/io/github/pedrofraca/tourapp/framework/database/StageDatabaseFactory.kt
package io.github.pedrofraca.tourapp.framework.database

import android.content.Context
import androidx.room.Room

object StageDatabaseFactory {
    fun getDatabase(context: Context?): TourDatabase {
        return Room.databaseBuilder(context!!, TourDatabase::class.java, "tour-db")
                .fallbackToDestructiveMigration()
                .build()
    }
}